.github/maintainer_institutes.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Dan Ellis": {"institute": "CMIP-IPO", "published_name": "Daniel Ellis"},
```

Comment:
are github handles easier to key off? Most contributors will only have their handle, with no more identifying info

Reply:
I considered this, but they do not come up within the `git log`s by default and I failed to get the `--format` options to work on my system.